builtin/logical/pki/path_tidy.go

```diff
@@ -178,7 +178,12 @@ func (b *backend) pathTidyWrite(ctx context.Context, req *logical.Request, d *fr
 						return errwrap.Wrapf(fmt.Sprintf("unable to parse stored revoked certificate with serial %q: {{err}}", serial), err)
 					}
 
-					if time.Now().After(revokedCert.NotAfter.Add(bufferDuration)) {
+					// Remove the matched certificate entries from revoked/ and
+					// cert/ paths. We compare against both the NotAfter time
+					// within the cert itself and the time from the revocation
+					// entry, and perform tidy if either one tells us that the
+					// certificate has already been revoked.
+					if time.Now().After(revokedCert.NotAfter.Add(bufferDuration)) || time.Now().After(revInfo.RevocationTimeUTC.Add(bufferDuration)) {
```

Comment:
Should we use `time.UTC().Now()` on this line?

Reply:
I don't think the location matters when comparing time objects, but I would recommend that we store `time.Now()` into a variable since we're using it twice.

Reply:
Yeah, time comparison via `After` should be based on absolute time so timezone does not really matter in this case. I'll store `time.Now()`, thanks for the suggestion!